src/main/java/ch/wisv/areafiftylan/products/service/MolliePaymentService.java

```diff
@@ -113,10 +113,18 @@ public Order updateStatus(String orderReference) {
                         order.setStatus(OrderStatus.PENDING);
                         break;
                     }
+                    case "open": {
+                        order.setStatus(OrderStatus.PENDING);
+                        break;
+                    }
                     case "cancelled": {
                         order.setStatus(OrderStatus.CANCELLED);
                         break;
                     }
+                    case "failed": {
+                        order.setStatus(OrderStatus.CANCELLED);
```

Comment:
Why not create a new enum `FAILED`?

Reply:
Wouldn't add anything of value for us. The same logic applies when an order is cancelled.